test/SourceKit/InterfaceGen/gen_clang_module.swift.response

```diff
@@ -5975,6 +5977,8 @@ public class FooOverlayClassDerived : Foo.FooOverlayClassBase {
         key.namelength: 10,
         key.attributes: [
           {
+            key.offset: 6061,
+            key.length: 15,
```

Comment:
no idea why this is 15 and not 4

Reply:
So, I think this is actually a `unowned(unsafe)` and according to `Attr.def`, `key.attribute: source.decl.attribute.weak` is used for `unowned` too. So this is probably the expected behavior.